src/main/java/com/client/core/base/util/Utility.java

```diff
@@ -802,32 +773,17 @@ private static <T extends SearchEntity> void searchForAll(Class<T> type, String
     }
 
     private static <T extends SearchEntity> void searchForAll(Class<T> type, String where, Set<String> fields, Consumer<List<T>> process, Integer start) {
-        BullhornData bullhornData = getBullhornData();
-
         SearchParams params = ParamFactory.searchParams();
         params.setStart(start);
         params.setCount(BATCH_SIZE);
         params.setSort("id");
 
-        ListWrapper<T> result = bullhornData.search(type, where, fields, params);
+        ListWrapper<T> result = BULLHORN_DATA.search(type, where, fields, params);
```

Comment:
This is so strange. Why is this changed at all? I wouldn't change something like this